File: web_shop/src/main/webapp/js/service/ordersService.js
//服务层
app.service('ordersService',function($http) {
//批量发货
    this.updateStatus = function(selectIds,status){
        return $http.get('../orders/updateStatus.do?selectIds='+selectIds+"&status="+status);
    }

//搜索
    this.search=function(page,rows,searchEntity){
        return $http.post('../orders/search.do?page='+page+"&rows="+rows, searchEntity);
    }
});

File: interface/src/main/java/cn/itcast/core/service/OrdersService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.entity.PageResult;
import cn.itcast.core.pojo.order.Order;

public interface OrdersService {

    PageResult findPage(Order ordersEntity, Integer page, Integer rows);

    void updateStatus(Order order, String status);
}
